.circleci/config.yml

```diff
@@ -77,53 +51,113 @@ jobs:
     - run:
-        name: build and test java
-        command: |
-          ./gradlew test --no-daemon
-
-      # disable until chrome binary downloads are fixed
-#    - run:
-#        name: test e2e
-#        command: |
-#          npx playwright install --with-deps
-#          npm install -D @playwright/test
-#          set CI=true && npx playwright test --config e2e --project=chromium
-
-
-    # to switch to release add following
-    # ./gradlew release jib -Dorg.ajoberstar.grgit.auth.username=${GITHUB_TOKEN} --stacktrace
-    #    ./gradlew checkFormat spotlessCheck test jacocoMergedReport sonarqube ci \
-    #    -Dsonar.login=${SONAR_TOKEN} -Dsonar.organization=molgenis -Dsonar.host.url=https://sonarcloud.io \
-    #    -Dorg.ajoberstar.grgit.auth.username=${GITHUB_TOKEN} -Dorg.ajoberstar.grgit.auth.password
+        name: install docker kubectl and helm
+        command: sh ci/install_docker_helm_kubectl.sh
+
+    - setup_remote_docker:
+        version: 19.03.13
+        docker_layer_caching: true
+
+    - run:
+        name: Sign in to docker
+        command: docker login -u $DOCKER_USER -p $DOCKER_PASS
+    # end install additional software
+
     - run:
-        name: release, push docker, push python
+        name: push emx2 docker images
         command: |
-          if [ "${CIRCLE_BRANCH}" = "master" ]; then
-            ./gradlew -s jacocoMergedReport --no-daemon shadowJar dockerPush helmPublishMainChart ci sonar release \
-            -Dorg.ajoberstar.grgit.auth.username=${GITHUB_TOKEN} -Dorg.ajoberstar.grgit.auth.password \
-            -Dsonar.login=${SONAR_TOKEN} -Dsonar.organization=molgenis -Dsonar.host.url=https://sonarcloud.io -Dsonar.verbose=true
-          else
-            echo "PR number: ${CIRCLE_PULL_REQUEST##*/}"
-            ./gradlew -s jacocoMergedReport --no-daemon shadowJar dockerPush helmPublishMainChart ci sonar release \
-            -Dorg.ajoberstar.grgit.auth.username=${GITHUB_TOKEN} -Dorg.ajoberstar.grgit.auth.password \
-            -Dsonar.login=${SONAR_TOKEN} -Dsonar.organization=molgenis -Dsonar.host.url=https://sonarcloud.io -Dsonar.verbose=true \
-            -Dsonar.pullrequest.key=${CIRCLE_PULL_REQUEST##*/} -Dsonar.pullrequest.branch=${CIRCLE_BRANCH}
-          fi
+          echo "PR number: ${CIRCLE_PULL_REQUEST##*/}"
+          ./gradlew -s jacocoMergedReport --no-daemon shadowJar dockerPush helmPublishMainChart ci sonar -x test -x :apps:buildJavascript -x :apps:checkFormat :apps:testJavaScript \
+          -Dorg.ajoberstar.grgit.auth.username=${GITHUB_TOKEN} -Dorg.ajoberstar.grgit.auth.password \
+          -Dsonar.login=${SONAR_TOKEN} -Dsonar.organization=molgenis -Dsonar.host.url=https://sonarcloud.io -Dsonar.verbose=true \
+          -Dsonar.pullrequest.key=${CIRCLE_PULL_REQUEST##*/} -Dsonar.pullrequest.branch=${CIRCLE_BRANCH}
           export $( cat build/ci.properties | xargs )
-          if [[ "$TAG_NAME" == *"SNAPSHOT"* ]]; then
```

Comment:
I cannot completely follow: do we now have seperate jobs for the 'snapshot' and 'release'?

Reply:
no same as it was before